packages/interface/src/components/ImageUpload.tsx

```diff
@@ -33,6 +33,10 @@ export const ImageUpload = ({
     },
   });
 
+  const onClickIconButton = useCallback((e: React.MouseEvent<HTMLButtonElement>) => {
+    e.preventDefault();
+  }, []);
```

Comment:
Why do you need this function?

Reply:
To prevent the form to be submitted (and show the validation errors) when clicking on the image

Reply:
to solve this problem: https://github.com/privacy-scaling-explorations/maci-platform/pull/398#issuecomment-2419999794